atomicfu/build.gradle

```diff
@@ -171,10 +208,12 @@ if (rootProject.ext.native_targets_enabled) {
             }
         }
 
-        configure(nativeCompilations) {
-            cinterops {
-                interop {
-                    defFile 'src/nativeInterop/cinterop/interop.def'
+        targets.withType(KotlinNativeTarget).configureEach { nativeTarget ->
```

Comment:
Could you please leave a comment in the code about why we need this block? As I understand from this PR description, we can remove it when [KT-60874](https://youtrack.jetbrains.com/issue/KT-60874/KLIB-Resolve-Dont-save-transitive-dependencies-in-manifests-depends-property) is done